servicenow-itsm.sgnl.yaml

```diff
@@ -9,8 +9,20 @@ defaultApiCallFrequency: SECONDLY
 defaultApiCallMinInterval: 1
 type: "ServiceNow-1.0.1"
 # Example Config:
-# {"requestTimeoutSeconds": 10, "apiVersion": "v2"}
-adapterConfig: "eyJyZXF1ZXN0VGltZW91dFNlY29uZHMiOiAxMCwgImFwaVZlcnNpb24iOiAidjIifQ=="
+#{
+#  "requestTimeoutSeconds": 10,
+#  "apiVersion": "v2",
+#  "filters": {
+#    "incident": "active=true",
```

Comment:
nit: We need not update the default adapter config but it'd be nice for the users to have some filtering syntax example with a helpful filter

```suggestion
#    "incident": "active=true^sys_updated_onRELATIVEGE@minute@ago@60",
```

Reply:
Oh I like that, I will in fact add it to the config too, I think - from https://docs.servicenow.com/bundle/washingtondc-platform-user-interface/page/use/common-ui-elements/reference/r_OpAvailableFiltersQueries.html

I believe, "Any Incident updated in the last 30 days" is:

"incident": "active=true^sys_updated_onRELATIVEGT@day@ago@30"

Is that your understanding too?

Reply:
That looks correct to me. I'd recommend we also test this filter against a live instance before we publish this.

Reply:
Ah yeah, sorry didnt update this comment -- tried day first, didn't work, switched over to dayofweek